Makefile

```diff
@@ -32,9 +32,9 @@ all: test
 
 .PHONY: clean
 clean:
-	-@find . -name "obj" | xargs rm -rf
-	-@find . -name "*.pyc" | xargs rm -rf
-	-@find . -name "*results.xml" | xargs rm -rf
+	-@find . -name "obj" -exec rm -rf {} +
```

Comment:
Huh, `-delete` doesn't work on directories. I love it when flags don't behave consistently. There doesn't seem to be a `-deletedir` option either. No action, just leaving justification for posterity.

Reply:
Now that I think of it `find . -depth \( -path "*/obj/*" -o -name "obj" \) -delete` would do it, but well...

I could not find "obj" in the hierarchy, this lines comes from the original commit, and I am not sure if it is a file or a directory.

Reply:
I'm not sure either. Better to just leave it be.